python/kvikio/_lib/nvcomp Cascade Benchmarks.ipynb

```diff
@@ -0,0 +1,3672 @@
+{
```

Comment:
Should we move this to `python/examples`?

Reply:
I was leaning toward moving it to `$KVIKIO_HOME/notebooks`, but I'm easily swayed.

Reply:
Sure that could make sense